pages/blog/the-reference-rabbit-hole.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "The Reference Rabbit Hole"
+date: 2021-11-15T12:00:00+01:00
+type: Engineering
+tags:
+  - Specification
+  - JSON Schema 
+cover: /img/posts/the-reference-rabbit-hole.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+---
+
+So [Sergio](https://github.com/smoya) and I, went down a little bit of a rabbit hole the last couple of days when we were discussing [Fran's proposal to solve publish/subscribe confusion](https://github.com/asyncapi/spec/issues/618), and I thought I would share the journey.
+
+## What is a reference?
+Just to start with the basics, a reference is something in AsyncAPI we use to allow for reusability, so we don't need to define things twice. In AsyncAPI we refer to this as the [Reference Object](https://www.asyncapi.com/docs/specifications/v2.2.0#referenceObject). The reference can either be local within the same document, external in another file or remotely.
```

Comment:
I would just add a short explanation of how references work. Just something like "you define once, and then you kind of "include" that definition wherever you want".
In that way, the reader will have a clear idea of what we talk about without having to check the link.

Reply:
Makes sense 👍